crawler/file_processing.py

```diff
@@ -409,7 +404,7 @@ def set_state_for_file(self) -> CentreFileState:
 
         # check for this being an unconsolidated samples file where the centre doesn't support those
         elif (
-            centre.get(CENTRE_KEY_SKIP_UNCONSOLIDATED_SURVEILLANCE_FILES, False)
+            self.centre_config.get(CENTRE_KEY_SKIP_UNCONSOLIDATED_SURVEILLANCE_FILES, False)
```

Comment:
OK am I understanding this correctly? So before your change it seems like some elements were being retrieved from the `centre_config` that was passed into the `Centre` class, while some were being retrieved from the database. After your change, they're all being retrieved from the `centre_config` that's passed in, but it looks to me like the only place this class is instantiated (outside tests) is in `main.py`, where you're passing in the centres from the db as the argument. So it's all actually consistently coming from the db now.

Is that right? If so seems like a really nice change 👍

Reply:
Yes I think what you described is the intended change.  No more looking up the database mid processing.  All entries come from the database and the lookup is one time when the run begins.